src/libs/getPersonalDetailsByIDs.js

```diff
@@ -0,0 +1,18 @@
+import Onyx from 'react-native-onyx';
```

Comment:
In the design doc we have this file being called `PersonalDetailsUtils.js`.

Additionally in ReportUtils we already have a method `getPersonalDetailsForLogin`. Is there a reason we decided not to put this method right there since it's effectively the same thing?

Reply:
renamed it to PersonalDetailsUtils.js

Reply:
is it fine this way?

Reply:
Whats the reasoning for putting these in ReportUtils? As far as I can tell they are not related to a Report they are related to PersonalDetails so makes sense they are their own utility but I don't really have a strong opinion here.